content/zh-cn/docs/concepts/cluster-administration/addons.md

```diff
@@ -84,8 +84,8 @@ Add-ons 扩展了 Kubernetes 的功能。
   CaaS / PaaS 平台（例如关键容器服务（PKS）和 OpenShift）之间的集成。
 * [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst)
   是一个 SDN 平台，可在 Kubernetes Pods 和非 Kubernetes 环境之间提供基于策略的联网，并具有可视化和安全监控。
-* Romana 是一个 pod 网络的第三层解决方案，并支持
-  [NetworkPolicy API](/zh/docs/concepts/services-networking/network-policies/)。
+* [Romana]((https://github.com/romana)) 是一个 Pod 网络的第三层解决方案，并支持
+  [NetworkPolicy](/zh/docs/concepts/services-networking/network-policies/) API。
   Kubeadm add-on 安装细节可以在[这里](https://github.com/romana/romana/tree/master/containerize)找到。
```

Comment:
This sentence is removed.

Reply:
no, here it is, https://github.com/kubernetes/website/blob/cc08154e70de230720cc793caf706a3612b58919/content/en/docs/concepts/cluster-administration/addons.md?plain=1#L33

Reply:
line 89 is removed...